src/store/message.ts

```diff
@@ -6,7 +6,7 @@ import { LS_KEYS } from 'src/constants';
 import create from 'zustand';
 import { persist } from 'zustand/middleware';
 
-type tabValues = 'Following' | 'Requested';
+type TabValues = 'Following' | 'Requested';
```

Comment:
It looked like most types start with a capital letter in the project so I bumped it!